test/coreputation/upsert_validator.js

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+const { AccountProvider } = require('../test_lib/utils.js');
+
+const Coreputation = artifacts.require('CoreputationTest');
+
+contract('Coreputation::upsertValidator', (accounts) => {
+  let accountProvider;
+  let coreputationInstance;
+  let inputValidatorInfo;
+  let coconsensus;
+  const ValidatorStatus = {
+    Undefined: 0,
```

Comment:
this pattern should be in `utils.js`

https://github.com/mosaicdao/mosaic-1/blob/develop/test/reputation/utils.js#L18

Reply:
Done!